pkg/kubelet/sysctl/allowlist_test.go

```diff
@@ -84,3 +91,60 @@ func TestAllowlist(t *testing.T) {
 		}
 	}
 }
+
+func TestAdmit(t *testing.T) {
+	tests := []struct {
```

Comment:
could you name the test cases? - it makes it easier for future readers to understand the intent of the test.

e.g

```
name: "admit pods that use kernel.shm_rmid_forced when not using HostIPC",
```

Reply:
updated, thanks.